app-backend/tile/src/main/scala/directives/TileAuthentication.scala

```diff
@@ -52,9 +52,7 @@ trait TileAuthentication extends Authentication
     parameter('mapToken).flatMap { mapToken =>
       val mapTokenId = UUID.fromString(mapToken)
 
-      val doesTokenExist = rfCache.caching(s"project-$projectId-token-$mapToken", 300) {
```

Comment:
I'm wondering if we should actually try and figure out a way to still keep this cached since will be happening for every shared map request